src/Turbo/Twig/FrameExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig helpers to generated the Turbo Frame elements.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * @experimental
+ */
+final class FrameExtension extends AbstractExtension
```

Comment:
After experimenting with this myself, I tend to think we don't actually need this extension. Using the HTML tags would allow developers to better understand the usage and I don't think Frames need Twig helpers. What do you think?

Reply:
I think we should provide a way to hide all Turbo(.js) related things. Even if it should be possible to also use the lower level HTML elements. IMHO it improves the DX to not have to know anything about Turbo to use the basic features (but maybe I'm wrong, I've no strong opinion about this.)

Reply:
I feel like writing `turbo_frame_start` or `<turbo-frame ...>` is basically similar in terms of not seing JS, but `<turbo-frame>` is easier to parse visually. I don't have strong opinion either. 

We could add the extension in the future if we want to though (much easier to add than remove)

Reply:
Having the tag is also more consistent with the Form extension. IMHO we should keep it, and we can remove it before marking the stable release if people don't use it (as it's entirely optional).

Reply:
After having discussed with Titouan and Ryan, I removed this extension to encourage using the HTML elements directly, as it's the new best practice we try to promote across the framework.